src/vs/workbench/contrib/files/common/explorerService.ts

```diff
@@ -155,7 +155,7 @@ export class ExplorerService implements IExplorerService {
 		}
 
 		// Stat needs to be resolved first and then revealed
-		const options: IResolveFileOptions = { resolveTo: [resource], resolveMetadata: false };
+		const options: IResolveFileOptions = { resolveTo: [resource], resolveMetadata: this.sortOrder === 'modified' ? true : false };
```

Comment:
given that `this.sortOrder === 'modified'` already returns a boolean, why do not we use that directly ?
```suggestion
		const options: IResolveFileOptions = { resolveTo: [resource], resolveMetadata: this.sortOrder === 'modified' };
```